syntax = "proto3";

package tensorflow;
option cc_enable_arenas = true;
option java_outer_classname = "RewriterConfigProtos";
option java_multiple_files = true;
option java_package = "org.tensorflow.framework";

message RewriterConfig {
  bool optimize_tensor_layout = 1;
  bool disable_model_pruning = 2;
  bool constant_folding = 3;
  // If non-empty, will use this as an alternative way to specify a list of
  // optimizations to turn on and the order of the optimizations.
  repeated string optimizers = 100;
}
